webserver/grpc/src/main/java/io/helidon/webserver/grpc/GrpcProtocolHandler.java

```diff
@@ -347,4 +396,45 @@ public MethodDescriptor<REQ, RES> getMethodDescriptor() {
             }
         };
     }
+
+    /**
+     * Initializes gRPC server metrics for the method being invoked. Note that
+     * duration and size metrics are currently recorded only for successful calls,
+     * using the {@link #OK_TAG}. If a call fails, it will only increment the number
+     * of started calls, but not record any of the other metrics.
+     */
+    private void initMetrics() {
+        if (grpcConfig.enableMetrics()) {
```

Comment:
Minor: `initMetrics` is not invoked unless `grpcConfig.enableMetrics()` has already evaluated to `true` in the caller so the `if` here is superfluous.

Reply:
Good point, will fix